README.md

```diff
@@ -1272,11 +1272,11 @@ While this guide explains the *what*, *why* and *how*, I find it helpful to see
 
 ###### [Style [Y076](#style-y076)]
 
-  - Use `bindToController = true` when using `controller as` syntax with a directive when you want to bind the outer scope to the directive's controller's scope.
+  - Use `bindToController` instead of `scope` when using `controller as` syntax with a directive when you want to bind the outer scope to the directive's controller's scope.
 
     *Why?*: It makes it easy to bind outer scope to the directive's controller scope.
 
-    Note: `bindToController` was introduced in Angular 1.3.0.
+    Note: `bindToController` was introduced in Angular 1.4.0.
```

Comment:
so this example will work only in 1.4. I wanted to keep it simple, instead of saying: bindToController was introduced in 1.3 but the object notation is only supported in 1.4